tests/unit/NavigationModelTest.php

```diff
@@ -22,117 +18,117 @@ function setUp() {
 	function testParseLines() {
 		$model = new NavigationModel();
 
-		$cases = array();
+		$cases = [];
 
-		$cases[] = array(
-			'param1' => array("*1", "*1", "*1", "**2", "**2", "****4"),
+		$cases[] = [
+			'param1' => [ "*1", "*1", "*1", "**2", "**2", "****4" ],
 			'param2' => null,
-			'out' => array(
-				0 => array(
-					'children' => array(1, 2, 3)
-				),
-				1 => array (
+			'out' => [
+				0 => [
+					'children' => [ 1, 2, 3 ]
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "children" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)